project/db/config.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+# Configuration goes here
+$config = array();
```

Comment:
It seems that you changed the database configuration file. 
Did you test it yet ? Since it is very sensitive code change.

Reply:
I tested on draft version so you can check using localhost and root as per host and login name